ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot

```diff
@@ -67,15 +67,20 @@ Setting Pod Toleration Via UI
 Verify RHODS Accept Multiple Admin Groups And CRD Gets Updates
     [Documentation]    Verify that users can set multiple admin groups and
     ...    check OdhDashboardConfig CRD gets updated according to Admin UI
-    [Tags]      Upgrade     RHOAIENG-14306    Platform
+    [Tags]      Upgrade     RHOAIENG-14306    Platform      RHOAIENG-19806
     [Setup]     Begin Web Test
     # robocop: disable
     Launch Dashboard And Check User Management Option Is Available For The User
     ...    ${TEST_USER.USERNAME}
     ...    ${TEST_USER.PASSWORD}
     ...    ${TEST_USER.AUTH_TYPE}
     Clear User Management Settings
-    Add OpenShift Groups To Data Science Administrators     rhods-admins        rhods-users
+    # Create a configmap and store both groups
+    ${return_code}    ${cmd_output}=    Run And Return Rc And Output
```

Comment:
You are using "${return_code}" and "${rc}" in the same change. I recommend using the same var name to avoid implied difference of functionality.

Reply:
changed to rc